server/routes/serviceProviderReferrals/showReferralPresenter.ts

```diff
@@ -57,9 +66,7 @@ export default class ShowReferralPresenter {
       { key: 'Complexity level', lines: [complexityLevelText.level, complexityLevelText.text], isList: false },
       {
         key: 'Date to be completed by',
-        lines: [
-          ShowReferralPresenter.govukFormattedDateFromStringOrNull(this.sentReferral.referral.completionDeadline),
-        ],
+        lines: [ViewUtils.govukFormattedDateFromStringOrNull(this.sentReferral.referral.completionDeadline)],
```

Comment:
What does this change have to do with the rest of the PR? If it's a refactor can you put it in a separate commit? But see later comment about suitability of `ViewUtils`.

Reply:
this is part of moving the banner code out of this presenter. will move the method from viewutils to the other class you suggested.